### ISSUE TITLE
Research practices / Coding standards

### ISSUE THREAD

**Issue by sjshim** [poldracklab/poldracklab.github.io#62]
- is python the preferred language?
- adding more tutorial resources?
- example of code that can be linked?
- tutorial / section about testing code (pytest etc.)

**Comment by sjshim** [comment]
guidance on using GPT?

**Comment by effigies** [comment]
> is python the preferred language?

IMO it's worth saying that Python is the most prevalent language, but some tasks are much better suited to R, especially on the side of statistics.

If we don't want to take a stand on "This is a Python lab" (and I don't think we should, but we kind of are and it's up to Russ to decide if that's official) it might be good to take one on "We use freely available, open source tools whenever possible."

> tutorial / section about testing code (pytest etc.)

FWIW I have a presentation from some years ago: https://effigies.github.io/testing-scientific-software/

**Comment by poldrack** [comment]
I agree - we should use whatever language makes the most sense given the particular problem. they should be open source languages (ie. no MATLAB or Stata) but I don't want to otherwise restrict what people use to do their work.
